common/constants/shared.ts

```diff
@@ -58,7 +58,7 @@ export const observabilityNotebookTitle = 'Notebooks';
 export const observabilityNotebookPluginOrder = 5094;
 
 export const observabilityPanelsID = 'observability-dashboards';
-export const observabilityPanelsTitle = 'Dashboards';
```

Comment:
can't change this here as it also changes in the name existing side-nav

Reply:
+1